src/test/components/__snapshots__/TooltipMarker.test.js.snap

```diff
@@ -1417,6 +1417,275 @@ exports[`TooltipMarker renders tooltips for various markers: FileIO (non-main th
+        <em
+          class="backtraceStackFrameOrigin"
+        />
+      </li>
+      <li
+        class="backtraceStackFrame"
+      >
+        mozilla::PresShell::HandleEventInternal
+        <em
+          class="backtraceStackFrameOrigin"
+        />
+      </li>
+      <li
+        class="backtraceStackFrame"
+      >
+        mozilla::PresShell::HandlePositionedEvent
+        <em
+          class="backtraceStackFrameOrigin"
+        />
+      </li>
+      <li
+        class="backtraceStackFrame"
+      >
+        mozilla::PresShell::HandleEvent
+        <em
+          class="backtraceStackFrameOrigin"
+        />
+      </li>
+      <li
+        class="backtraceStackFrame"
+      >
+        nsViewManager::DispatchEvent
+        <em
+          class="backtraceStackFrameOrigin"
+        />
+      </li>
+      <li
+        class="backtraceStackFrame"
+      >
+        nsView::HandleEvent
+        <em
+          class="backtraceStackFrameOrigin"
+        />
+      </li>
+      <li
+        class="backtraceStackFrame"
+      >
+        nsChildView::DispatchEvent
+        <em
+          class="backtraceStackFrameOrigin"
+        />
+      </li>
+      <li
+        class="backtraceStackFrame"
+      >
+        ChildViewMouseTracker::MouseMoved
+        <em
+          class="backtraceStackFrameOrigin"
+        />
+      </li>
+      <li
+        class="backtraceStackFrame"
+      >
+        nsAppShell::Run
+        <em
+          class="backtraceStackFrameOrigin"
+        />
+      </li>
+      <li
+        class="backtraceStackFrame"
+      >
+        nsAppStartup::Run
+        <em
+          class="backtraceStackFrameOrigin"
+        />
+      </li>
+      <li
+        class="backtraceStackFrame"
+      >
+        XREMain::XRE_mainRun
+        <em
+          class="backtraceStackFrameOrigin"
+        />
+      </li>
+      <li
+        class="backtraceStackFrame"
+      >
+        XREMain::XRE_main
+        <em
+          class="backtraceStackFrameOrigin"
+        />
+      </li>
+      <li
+        class="backtraceStackFrame"
+      >
+        XRE_main
+        <em
+          class="backtraceStackFrameOrigin"
+        />
+      </li>
+      <li
+        class="backtraceStackFrame"
+      >
+        _main
+        <em
+          class="backtraceStackFrameOrigin"
+        />
+      </li>
+    </ol>
+  </div>
+</div>
+`;
+
+exports[`TooltipMarker renders tooltips for various markers: FileIO (non-profiled thread)-114 1`] = `
+<div
+  class="tooltipMarker propClass"
+>
+  <div
+    class="tooltipHeader"
+  >
+    <div
+      class="tooltipOneLine"
+    >
+      <div
+        class="tooltipTiming"
+      >
+        1.0ms
+      </div>
+      <div
+        class="tooltipTitle"
+      >
+        FileIO (non-profiled thread)
+      </div>
+    </div>
+    <div
+      class="tooltipDetails"
+    >
+      <div
+        class="tooltipLabel"
+      >
+        Thread
+        :
+      </div>
+      Main Thread
```

Comment:
Isn't it misleading that we still display the thread where the marker is here? Should we display the thread information from the FileIO here instead than below in the payload, if present?

Reply:
The first `Thread` section is more generic and I feel like we shouldn't handle the specific payload cases there. The second section is payload specific and this is where I added this. Also it would be good to show the thread the marker actually belongs at the moment. What do you think?